src/lib/encoder/Decoder.h

```diff
@@ -125,3 +198,30 @@ inline unsigned Decoder::decode(std::string filename, std::string output)
 	std::ofstream f(output);
 	return decode(img, f, false);
 }
+
+inline bool Decoder::load_ccm(std::string filename)
+{
+	File f(filename);
+	std::string data = f.read_all();
+	if (data.size() < 3*3*4)
+		return false;
+
+	cv::Mat temp(3, 3, CV_32F, data.data());
```

Comment:
These save/load functions are not intended for anything beyond manual debugging/testing/verification at present. (this is for the two-pass "fountain decode, then non-fountain decode, then diff to check error rates" use case).